### ISSUE TITLE
Testing enum?

### ISSUE THREAD

**Issue by Bartuz** [thetron/mongoid-enum#26]
How should I test enum setup in my app?

**Comment by thetron** [comment]
@Bartuz The library is tested - so you don't need to test any of the features that you get from mongoid-enum, so I would just be testing your enums the same way you would test any attribute on your Mongoid documents.

**Comment by Bartuz** [comment]
Ok, but there are [rspec matchers](https://github.com/mongoid-rspec/mongoid-rspec) which I can use this way:
`  it { is_expected.to have_fields(:birthdate, :registered_at).of_type(DateTime) }`

there is not anything likee this for this gem, right?

**Comment by thetron** [comment]
No, the library doesn't come with any RSpec matchers, but the fields are (at this point in time) just strings (or symbols - I can't quite remember precisely), so:

```
it { is expected.to have_fields(:foobar_enum).of_type(String)
```

Would be adequate here, IMHO.
